main.cpp

```diff
@@ -48,27 +44,34 @@ int main()
 
     oep->load_effect("effects/Afro");
 
-    // Create and run instance of camera, pass callback for frames
-    // Callback for received frame from the camera
-    auto ef_cb = [&oep, render_t](pixel_buffer_sptr image) {
-        // Callback for received pixel buffer from the offscreen effect player
-        auto get_pixel_buffer_callback = [render_t](image_processing_result_sptr result) {
-            if (result != nullptr) {
-                // Callback for update data in render thread
-                auto render_callback = [render_t](std::optional<rendered_texture_t> texture_id) {
-                    if (texture_id.has_value()) {
-                        auto gl_texture = static_cast<GLuint>(reinterpret_cast<int64_t>(*texture_id));
-                        render_t->update_data(gl_texture);
-                    }
-                };
-                // Get texture id from shared context and render it
-                result->get_texture(render_callback);
-            }
+    auto camera_callback = [&oep, render_t](bnb::full_image_t image) {
+        // Create and run instance of camera, pass callback for frames
+        // Callback for received frame from the camera
```

Comment:
this comment is also looks that it is not in required place... it should be before this callback declaration